addons/web/static/src/js/views/search_panel.js

```diff
@@ -26,35 +26,43 @@ odoo.define("web/static/src/js/views/search_panel.js", function (require) {
             });
 
             this.state = useState({
-                temp: {},
+                active: {},
                 expanded: {},
             });
             this.model = useModel("searchModel");
+            // this.env.exportable({
+            //     export: this.exportState.bind(this),
+            //     import: this.importState.bind(this),
+            // });
+        }
 
+        async willStart() {
             this._expandDefaultValue();
+            this._updateActiveValues();
         }
 
         mounted() {
             this._updateGroupHeadersChecked();
-            if (this.props.scrollTop) {
-                this.el.scroll({ top: this.props.scrollTop });
-            }
         }
 
-        willUpdateProps(newProps) {
-            this.state.temp = {};
-            if ("expanded" in newProps) {
-                this.state.expanded = {};
-                for (const categoryId in newProps.expanded) {
-                    this.state.expanded[categoryId] = Object.assign(
-                        {}, newProps.expanded[categoryId]
-                    );
-                }
-            }
+        willUpdateProps() {
+            this._updateActiveValues();
+        }
+
+        //---------------------------------------------------------------------
+        // Public
+        //---------------------------------------------------------------------
+
+        exportState() {
+            return {
+                expanded: this.state.expanded,
+                scrollTop: this.el.scrollTop,
+            };
         }
 
-        willUnmount() {
-            clearTimeout(this.scrolling);
+        importState(state) {
+            this.state.expanded = state.expanded;
+            this.el.scroll({ top: this.props.scrollTop });
```

Comment:
I find this line weird

Reply:
Why ? (it has been moved to `mounted` unchanged)